src/gtimelog/timelog.py

```diff
@@ -1118,15 +1121,34 @@ def load(self):
                     if ':' in line:  # tasks with group prefix
                         group, task = [s.strip() for s in line.split(':', 1)]
                         groups.setdefault(group, []).append(task)
+                        tasks.append(group + ': ' + task)
                     else:  # "other" tasks
-                        others.append(line.strip())
+                        others.append(line)
+                        tasks.append(line)
```

Comment:
This `for line in f:` would become `for index, line in enumerate(f)` and you'd do `task_order[line] = index` here.

Reply:
Yep. done.